sound/soc/sof/debug.c

```diff
@@ -17,6 +17,167 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_IPC_FLOOD_TEST)
+#define MAX_IPC_FLOOD_DURATION_MS 1000
+#define MAX_IPC_FLOOD_COUNT 10000
```

Comment:
@ranj063 
I suggest to increase this number for stress test.
I test with and without #943 and find it is good for test. But it just test to little from my view
The 10000 times IPC only takes 410ms, this is not enough for stress test.

```
[33766.247317] sof-audio-pci 0000:00:0e.0: IPC Flood test count: 10000, Avg response time: 41090ns
[33766.247319] sof-audio-pci 0000:00:0e.0: IPC Flood test Max response time: 170337ns
[33766.247320] sof-audio-pci 0000:00:0e.0: IPC Flood test Min response time: 34170ns
```

Reply:
> I suggest to increase this number for stress test.

@xiulipan I consciously limited this to 10000. I know this is small but we can always write a script to loop over any number of times to repeat the 10K test. I will submit a script for this as well. What do you think?